lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -58,11 +58,9 @@ export class PublicClientApplication extends ClientApplication implements IPubli
         } else if (this.config.system.brokerOptions.allowBrokering) {
             this.embeddedApp = new EmbeddedClientApplication(this.config, this.logger, this.browserStorage);
             this.logger.verbose("Acting as child");
-            try {
-                this.embeddedApp.initiateHandshake();
-            } catch (e) {
+            this.embeddedApp.initiateHandshake().catch((e) => {
```

Comment:
does this need to be awaited?

Reply:
It does, but not here. PR #2300 exposes this as a public API which we will expect people to call and await in their app rather than doing async stuff in the constructor